pkg/util/files_test.go

```diff
@@ -0,0 +1,23 @@
+package util
+
+import (
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestContentTypeForFile(t *testing.T) {
+	t.Parallel()
+
+	testData := map[string]string{
+		"foo.log": "text/plain; charset=utf-8",
+		"foo.txt": "text/plain; charset=utf-8",
+		"foo.xml": "application/xml",
+		"foo.json": "application/json",
```

Comment:
undefined in some configurations

Reply:
maybe we should disable the test on windows etc?

Reply:
not just windows - it could be changed in different distributions.
(a chinese linux distrub may use a differernt default charset for txt files?)

is it important that these actually match?  if so then I think a map should be used - and use `mime` as a fallback.
if the result is not important  - then why test it?